FJ-about-smoke-spec.js

```diff
@@ -1,19 +1,58 @@
-describe('The new consumerfinance.gov', function() {
+describe( 'Beta Contact Page', function() {
+  var passedURL = browser.params.base_url;
+  var baseURL = passedURL ? passedURL : 'http://beta.consumerfinance.gov/';
+  var contactURL = baseURL + 'contact-us/';
+  var phoneClass = 'list_link__phone';
 
-  beforeEach(function() {
+  beforeEach( function() {
     return browser.ignoreSynchronization = true;
-  });
+  } );
 
-  it('should properly load in a browser', function(){
-    browser.get('http://beta.consumerfinance.gov/'); // This could be set to a staging server or localhost.
-    expect(browser.getTitle()).toBe('Consumer Financial Protection Bureau');
-  });
+  it( 'should properly load in a browser', function(){
+    browser.get( contactURL ); // This could be set to a staging server or localhost.
+    expect( browser.getTitle() ).toBe( 'Contact us' );
+  } );
 
-  it('should show the correct phone number', function(){
-    browser.get('http://beta.consumerfinance.gov/contact-us');
-    var phoneNumber = '(855) 411-CFPB (2372)'
-    var phoneElement = element(by.partialLinkText(phoneNumber));
-    expect(phoneElement.getText()).toBeDefined();
-  });
+  it( 'should include the Submit a Complaint phone numbers', function(){
```

Comment:
Some purists might argue a test should only be testing one condition. If the code to output these phone numbers on the page can be adjusted independently of each other it should be two tests, if not, then this is probably fine.